public/server/db.json

```
@@ -109,6 +109,57 @@
     },
     "ootd":{
         "definition": "Outfit Of The Day"
-    }
-    
+    },
+    "cmiiw":{
+        "definition": "Correct me if i'm wrong"
+    },
+    "b2w":{
+        "definition": "Back to work"
```

The first letter of each word should be in capital letters. Do this for every new abbreviation you added.
```suggestion
        "definition": "Back To Work"
```

---

Sorry about that, I thought I worked on it all. I will do that now